cms/djangoapps/cms_user_tasks/tests.py

```diff
@@ -102,6 +103,7 @@ def test_artifact_list(self):
         serializer = ArtifactSerializer(self.artifact, context=_context(response))
         assert _data(response)['results'] == [serializer.data]
 
+    @unittest.skip("Due to this test build never get green. Workers stuck without any failure.")
```

Comment:
This test is causing failure

Reply:
on `.post` it hangs.

Reply:
https://github.com/edx/django-user-tasks/blob/0c732facbd05a3505a97d2fbe72d73a5fde537f1/user_tasks/models.py#L126

Reply:
Its happening because of .revoke method called during test.
`current_app.control.revoke(self.task_id)`
https://github.com/edx/django-user-tasks/blob/9774f8ea7f293998fb58736e857b36053f3d4bc4/user_tasks/models.py#L135